src/Silex/EventListener/ConverterListener.php

```diff
@@ -39,12 +40,8 @@ public function __construct(RouteCollection $routes, CallbackResolver $callbackR
         $this->callbackResolver = $callbackResolver;
     }
 
-    /**
-     * Handles converters.
-     *
-     * @param FilterControllerEvent $event The event to handle
-     */
-    public function onKernelController(FilterControllerEvent $event)
+
+    public function onKernelController(ControllerEvent $event)
```

Comment:
From:  Symfony\Component\HttpKernel\Event
https://github.com/symfony/http-kernel/blob/v4.4.50/Event/FilterControllerEvent.php

```
/**
 * @deprecated since Symfony 4.3, use ControllerEvent instead
 */
```